Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8.1.4
+
+MAINTAINER "HMCTS Reform"
+LABEL maintainer="HMCTS Reform"
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
```

Comment:
~is package.json enough?~

p.s. since workdir is defined, no need to explicitly say where to copy things. but that's just char count - good to read the whole path